### ISSUE TITLE
Problem loading 2.0.0 on 64 bit Ubuntu 12.04

### ISSUE THREAD

**Issue by LoonyPandora** [LoonyPandora/Dancer-Plugin-Passphrase#8]
Hi --
I tried cpanm --sudo Task::Dancer
on 64bit Ubuntu 12.04 LTS and got the following error:

```
  Failed test 'SHA-1 needs encoded text'
  at t/008_unicode_matching.t line 20.
                  ''
    doesn't match '(?^i:Wide character in subroutine entry)'

  Failed test 'SHA-256 needs encoded text'
  at t/008_unicode_matching.t line 28.
                  ''
    doesn't match '(?^i:Wide character in subroutine entry)'
Looks like you failed 2 tests of 8.
```

This is using the system default perl

```
$ perl -v

This is perl 5, version 14, subversion 2 (v5.14.2) built for 
x86_64-linux-gnu-thread-multi
```

(I can't load Task::Dancer in perlbrew because some of the dependencies 
can't build 64-bit.)

Elyse Grasso

**Comment by ollyg** [comment]
Same as issue #7 you need to update Digest::SHA to at least 5.74.

**Comment by LoonyPandora** [comment]
This should be fixed by v2.0.1 released today: https://metacpan.org/release/JAITKEN/Dancer-Plugin-Passphrase-2.0.1

(sorry for the delay)
